modules/admin/src/views/layouts/main.php

```diff
@@ -104,13 +104,33 @@
                             </span>
                         </span>
                     </li>
-                    <li class="mainnav-entry" tooltip tooltip-text="<?= Admin::t('layout_btn_profile'); ?>" tooltip-position="right" tooltip-disabled="isHover">
-                        <a class="mainnav-link" ui-sref="custom({templateId:'admin/account/dashboard'})" ng-click="isOpen=0">
+                    <li class="mainnav-entry">
+                        <!-- needs to be fixed -->
+                        <span class="mainnav-parent" active-class="mainnav-parent-active" ng-mouseover="active(true);" ng-mouseout="active(false);">
```

Comment:
@dev7ch why using mouseover and mouseout when alread trigger trough active-class? what is the function active(true) which is called?

Reply:
https://github.com/dev7ch/luya/blob/6d464561ea78544fb10faf98398a3c8256bf51b5/modules/admin/src/resources/js/directives.js#L3021

Reply:
oh i see it can be removed.